src/Job/Import.php

```diff
@@ -204,12 +204,12 @@ public function resourceToJson(EasyRdf_Resource $resource)
     /**
      * Get the property id for an rdf property, if known in Omeka
      *
-     * @param string or EasyRdf_Resource $property
+     * @param string or RdfResource $property
```

Comment:
This is a preexisting issue but we might as well fix it if you're in here: the proper way to write this would be `string|RdfResource`